proxy/papyrus/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM ghcr.io/starkware-libs/papyrus:dev
+
+USER root
```

Hadolint

[warning] Last USER should not be root

(DL3002)